cmd/samToWig/testdata/test2.wig

```diff
@@ -2,31 +2,6 @@ fixedStep chrom=chr1 start=1 step=1
 0
 0
 0
-0
```

Comment:
I didn't look into the details, but I am surprised that test2.wig is changing (similar to test2.bed above where I did look into it quick).  I wouldn't think this output is dependent on the dna seq in the sam